bindings/python/tersets/__init__.py

```diff
@@ -67,8 +68,16 @@ class __Configuration(Structure):
     _fields_ = [("method", c_byte), ("error_bound", c_float)]
 
 
+# Mirror TerseTS Method Enum    
```

Comment:
Missing punctuation at the end of the comment.

Reply:
Fixed.